src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -114,6 +119,16 @@ Version VSVersion
 
         protected override async tasks.Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
+            var environment = new Rothko.Environment();
+            var logPath = Path.Combine(environment.GetLocalGitHubApplicationDataPath(), "extension.log");
+
+            Log.Logger = new LoggerConfiguration()
+                .Enrich.WithThreadId()
+                .WriteTo.File(logPath,
+                    fileSizeLimitBytes: 2L * 1024L * 1024L,
+                    outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss.fff}|{Level}|Thread:{ThreadId}|{SourceContext}|{Message}{NewLine}{Exception}")
+                .CreateLogger();
+
```

Comment:
This is the earliest entry point I could find that allowed me to configure serilog.
It has to be done before `UIProvider` is statically constructed.

I'm not sure this is where this code should stay permanently.